src/pages/Login.tsx

```
@@ -1,22 +1,63 @@
+      newErrors.password = '비밀번호를 입력해주세요'
+    }
+    setErrors(newErrors)
+    return Object.values(newErrors).every((error) => error === '')
+  }
+
+  const handleLogin = async () => {
+    if (validation()) {
+      try {
+        await login(form)
+          .then(() => navigate('/main'))
+          .catch((error) => setErrors((prev) => ({ ...prev, password: '로그인에 실패하였습니다' })))
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)